### DISCUSSION TITLE
Orchid Platform 12.x and Tenancy for Laravel 3.x

### DISCUSSION THREAD

**Discussion by Impeck** [orchidsoftware/platform#2197] [Q&A]
It's about **stancl/tenancy** 3.x, quickstart tutorial https://tenancyforlaravel.com/docs/v3/quickstart/

There is a problem with basic authorization Orchid. 

For example, central domain **orchid.local** and a tenancy subdomain **foo.orchid.local**.
There is no middleware tenancy for route on the subdomain **foo.orchid.local/admin/login**, the request goes to the central database, the user on the tenancy database cannot log in.

In the example here https://dev.to/ostap/orchid-platform-meets-multi-tenancy-4e10 Tenancy solved differently (basic authorization is not used).

How to correctly configure the Orchid Platform and Tenancy for Laravel? How to set up routing correctly for **stancl/tenancy?**. Thanks.

**Comment by ContendoEmp** [comment]
Hello,

I simply had to override the route by duplicating it in my tenant route file

    Route::post('/admin/logout', [LoginController::class, 'logout'])->name('platform.logout');

I created my login controller that contains the exact code from the orchid login controller
